bazel/test_runners/qemu_with_kernel/run_qemu.sh

```diff
@@ -76,6 +76,13 @@ flags+=(-append "console=ttyS0 root=/dev/sda")
 # Disable graphics mode.
 flags+=(-nographic)
 
+# Change to the OVMF bios if it exists.
+# This fixes issues with terminal/screen cocrruption after running qemu.
+ovmf_bios=/usr/share/ovmf/OVMF.fd
```

Comment:
@zasgar what do you think about installing this with our chef code so anyone using the standard development flow avoids the terminal corruption?